srcpkgs/libxdg-basedir/template

```diff
@@ -1,15 +1,21 @@
 # Template file for 'libxdg-basedir'
 pkgname=libxdg-basedir
 version=1.2.0
-revision=3
+revision=4
+_githash=86ea74ee174c6bee204729293b2b292492d35cb3
```

Comment:
I think that is good in general, but in this case, the only significant update to the libxdg-basedir git repo since 1.2.0 (which is 6 years ago) was this bug fix. So I think it is fine to use the git revision.